src/ui/public/notify/__tests__/notifier.js

```diff
@@ -193,7 +259,7 @@ describe('Notifier', function () {
     });
 
     it('prepends location to message for markdown', function () {
-      expect(notify('banner').markdown).to.equal(params.location + ': ' + message);
+      expect(notify('banner').content).to.equal(params.location + ': ' + message);
```

Comment:
This can be improvement in a small way with ES2015 template literals: `${params.location}: ${message}`